web/scss/_layout.scss

```diff
@@ -0,0 +1,141 @@
+@include font-face(200, "../typefaces/inter", "Inter UI");
+@include font-face(300, "../typefaces/inter", "Inter UI");
+@include font-face(400, "../typefaces/inter", "Inter UI");
+@include font-face(700, "../typefaces/inter", "Inter UI");
+
+html,
+body {
+  width: 100%; height: 100%;
+}
+
+html {
+  background-color: $lbry-white;
+}
+
+body {
+  @include font-sans;
+  display: flex;
+  flex-direction: column;
+  font-size: 12px;
+}
+
+
+
+code {
+  @include font-mono;
+}
+
+h1,
```

Comment:
Is there a shared scope it would make sense for some of these rules (particularly the header stylings) can go in rather than being global? I suspect a number of custom pages or elements would want to present an h3 differently.

Reply:
That makes sense, I could move these to `.ancillary` for the time being.